src/main/webapp/js/courseSelector.js

```diff
@@ -98,15 +135,30 @@ export const CollegePlanner = (() => {
     return liElement;
   }
   window.addEventListener('load', () => {
-    getOptions();
+    getDepartmentOptions();
   });
   document.getElementById('add-selected').addEventListener('click', () => {
     addToSelected();
   });
+  document.getElementById('departments').addEventListener('change', () => {
+    getCourseOptions();
+  });
 
+  function getSelected(){
```

Comment:
nit: missing space between `()` and `{`. Is the linter broken / not firing for these?

Reply:
nope, I think I just forgot to run the linter before my last commit, thanks for pointing this out.